src/github.rs

```diff
@@ -1825,20 +1825,34 @@ impl<'q> IssuesQuery for Query<'q> {
                         issue.html_url, fcp.fcp.fk_bot_tracking_comment
                     );
                     let bot_tracking_comment_content = quote_reply(&fcp.status_comment.body);
-
                     let fk_initiating_comment = fcp.fcp.fk_initiating_comment;
-                    let init_comment = issue
-                        .get_comment(&client, fk_initiating_comment.try_into()?)
-                        .await?;
+                    let (initiating_comment_html_url, initiating_comment_content) =
+                        if u32::try_from(fk_initiating_comment).is_err() {
+                            // We blew out the GH comment incremental counter (a i32 on their end)
+                            // See: https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/rfcbot.20asleep
+                            log::debug!("Ignoring overflowed comment id from GitHub");
+                            ("".to_string(), "".to_string())
```

Comment:
I don't fully understand why this is using empty strings. But I'll trust since I assume you do most of the agenda stuff that it works for you.

Reply:
yes it's a bit of a hack and I am not either very fond of this. `initiating_comment_html_url` and `initiating_comment_content` are mandatory fields in `FCPDetails`, so the minimal change I could think of was to set them to an empty string rather than making them `Option<String>` and deal with a widerspread change in the codebase.